docs/learn/response_models.md

```diff
@@ -35,33 +35,38 @@ print(f"Title: {book.title}")
 print(f"Author: {book.author}")
 ```
 
+In this example:
+
+1. We define a `Book` class that inherits from Pydantic's `BaseModel`.
+2. We use the `@openai.call` decorator with the `response_model` parameter set to our `Book` class.
+3. The `recommend_book` function is decorated with both `@openai.call` and `@prompt_template`.
+
 This approach works consistently across all supported providers in Mirascope.
 
-!!! tip "Original Response"
+!!! tip "Accessing the Original Response"
 
     You can always access the original response through the `_response` property on the returned response model. However, this is a private attribute we've included for such access and is not an official property of the response model type, meaning that you'll need to ignore type errors when accessing the property and cast the property to get editor support on the original response object.
 
     Example:
     
-    ```python
+    ```python hl_lines="2 4"
     # original `OpenAICallResponse`
     response = cast(openai.OpenAICallResponse, book._response)  # pyright: ignore [reportAttributeAccessIssue]
     # original `ChatCompletion`
     completion = response.response
     ```
 
-### Extracting Built-in Types
+    Note that this is a private attribute included for access to the original response and is not an official property of the response model type.
```

Comment:
nit: this is duplicated from the above explanation before the code snippet in the note?